<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingsWindow</class>
 <widget class="QWidget" name="settingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>921</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Speech control by Igor Pro Settings</string>
  </property>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>860</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Команды добавления</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delete_minus">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>370</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>640</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Команда остановки</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Команды удаления</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_add">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>200</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_add_minus">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>200</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_add">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>100</y>
     <width>191</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_start">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_delete">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>270</y>
     <width>191</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_stop">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>650</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_add_plus">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>200</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delete_plus">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>370</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Команда запуска прослушивания</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_delete">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>370</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>700</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Максимальная пауза между словами</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>710</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="minimum">
    <double>0.500000000000000</double>
   </property>
   <property name="maximum">
    <double>4.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_search_minus">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>540</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_search">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>440</y>
     <width>191</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Команды поиска</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_search">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>540</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_search_plus">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>540</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>770</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Скорость озвучивания</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_speed">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>790</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string>Обычная</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>Обычная</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ускорение 1.5x</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ускорение 2.0x</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ускорение 2.5x</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>590</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Команда аренды</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_rent">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>600</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
